dspace-oai/src/main/java/org/dspace/utils/LangUtil.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ * 
+ * http://www.dspace.org/license/
+ */
+
+/* Created for LINDAT/CLARIAH-CZ (UFAL) */
+package org.dspace.utils;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+
```

Comment:
doc

Reply:
there is doc now, please comment or resolve this thread